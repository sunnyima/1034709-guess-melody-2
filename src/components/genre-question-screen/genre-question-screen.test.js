import React from 'react';
import renderer from 'react-test-renderer';
import GenreQuestionScreen from "./genre-question-screen";

it(`GenreQuestionScreen correctly renders after relaunch`, () => {
  const tree = renderer.create(<GenreQuestionScreen
    screenIndex={0}
    question={{
      answers: [],
      genre: ``,
    }}
    onAnswer={ jest.fn() }
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
