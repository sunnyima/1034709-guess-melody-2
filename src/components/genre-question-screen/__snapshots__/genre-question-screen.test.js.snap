// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreQuestionScreen correctly renders after relaunch 1`] = `
<section
  className="game game--genre"
>
  <header
    className="game__header"
  >
    <a
      className="game__back"
    >
      <span
        className="visually-hidden"
      >
        Сыграть ещё раз
      </span>
      <img
        alt="Угадай мелодию"
        className="game__logo"
        src="img/melody-logo-ginger.png"
      />
    </a>
    <div
      className="timer__value"
    >
      <span
        className="timer__mins"
      >
        05
      </span>
      <span
        className="timer__dots"
      >
        :
      </span>
      <span
        className="timer__secs"
      >
        00
      </span>
    </div>
    <div
      className="game__mistakes"
    >
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
    </div>
  </header>
  <section
    className="game__screen"
  >
    <h2
      className="game__title"
    >
      Выберите 
      
       треки
    </h2>
    <form
      className="game__tracks"
      onSubmit={[Function]}
    >
      <button
        className="game__submit button"
        type="submit"
      >
        Ответить
      </button>
    </form>
  </section>
</section>
`;
